index.html

```diff
@@ -63,5 +63,187 @@ <h1 class="narson">I'm Narson.
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="buttons">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <!--About me section-->
+
+    <section class="about-me">
+        <div class="about-container">
+        <h3 class="about-text">About Myself</h3>
+        <p class="summary">
+            Hello I’m a software developer! I can help you build a product , feature or website. 
+            Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <h4 class="connection2">LET'S CONNECT</h4>
+
+        <ul class="social2">
+            <li class="twitter">
+                <a href="https://twitter.com/@Narson321">
+                <img src="media/twitter.svg" alt="twitter" title="twitter"/>
+                </a>
+            </li>
+            <li class="linkindin">
+                <a href="https://linkedin.com/in/andry-narson-rabedesana-15b8b4248">
+                    <img src="media/Linkedin icon.svg" alt="Linkindin" title="linkindin"/>
+                </a>
+            </li>
+            <li class="github">
+                <a href="https://github.com/rbd3">
+                <img src="media/github.svg" alt="github" title="github" />
+                </a>
+            </li>
+            <li class="angellist"><a href="#">
+                </a>
+                <img src="media/angellist.svg" alt="Angellist" title="Angellist" />
+                </a>
+            </li>
+            <li class="medium">
+                <a href="#">
+                <img src="media/medium.svg" alt="Medium" title="Medium" />
+                </a>
+            </li>
+        </ul>
+        <button type="button" class="buttons2">Get my resume</button>
+        </div>
+
+        <div class="skill-list">
+            <p class="languages">Languages</p>
+            <img src="media/Icon - Chevron - Down - Larger.svg" alt="Down Arrow" class="down-arrow" />
+
+            <div class="languages-container">
+                <ul class="languages-list2">
+                    <li class="javascript2">
+                       <img class="js-logo" src="media/js.svg" alt="Javascript logo" />
+                       <p class="js-text">JavaScript</p> 
+                    </li>
+                    <li class="html2">
+                        <img class="html-logo" src="media/html.svg" alt="HTML logo" />
+                        <p class="html-text">HTML</p> 
+                     </li>
+                     <li class="css2">
+                        <img class="css-logo" src="media/css.svg" alt="CSS logo" />
+                        <p class="css-text">CSS</p> 
+                     </li>
+                </ul>
```

Comment:
Kindly reduce the width of each technology to match the figma design better🙏

Your design:

<img width="298" alt="image" src="https://user-images.githubusercontent.com/37116322/235078914-fbc84780-a841-4e9c-b686-1961ae4a4a90.png">

Figma design:

<img width="341" alt="image" src="https://user-images.githubusercontent.com/37116322/235078868-3412f815-e705-4ee8-9f9d-8d9685138194.png">